src/Migration.php

```diff
@@ -240,7 +240,7 @@ public function displayError(string $message): void
     /**
      * Define field's format
      *
-     * @param string  $type          can be bool, char, string, integer, date, datetime, text, longtext or autoincrement
+     * @param string  $type          can be bool, char, string, int, date, time, datetime, text, autoincrement, fkey, json
```

Comment:
`integer` is valid, `longtext` as well... Why replacing `integer` with `int` and not `string` with `str`?
Maybe should we just simplify this entry and do not try to maintain a complex list.

Reply:
we must agree on the documented case.
should we use short form (bool, int, etc) or long form (boolean, integer).
at the moment, it's mixed.

Reply:
This is not the same here. It documents the expected values of the `$type` parameter of the `Migration::fieldFormat()`.